src/python/pants/help/help_info_extracter.py

```diff
@@ -1,16 +1,17 @@
 # Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from collections import namedtuple
+import inspect
+from dataclasses import dataclass
+from typing import List, Optional, Type
 
 from pants.base import deprecated
 from pants.option.option_util import is_list_option
+from pants.util.collections import Enum
```

Comment:
You'll need to add to the `BUILD` file for this target a dependency on `pants/util/collections:enum`. Otherwise, `./pants help native-build-step` fails to run.

Reply:
Thanks! I'm assuming you meant `src/python/pants/util:collections` tho?

Reply:
Oh oops, yes, I did :)